docs/start/smesher/trusted_state_quicksync.md

```diff
@@ -0,0 +1,45 @@
+---
+id: trusted_state_quicksync
+title: Trusted State aka Quicksync
+---
+
+## Quicksync
+
+In response to the growing difficulty of syncing a fresh node, we have prepared a way to speed up the syncing process. Introducing, Quicksync.
+
+Before reading any further, plaese read the annoucement about Quicksync [here](https://spacemesh.io/blog/making-sync-faster/).
+
+### Getting the archive automatically
```

Comment:
It mentions "the archive" without explaining what it is, causing confusion about whether it is an archive of the quicksync tool itself or something else.

Reply:
Thanks for the feedback, @poszu! Agree, the intro should explain the issue and the solution a bit. I have written the following introduction that will come under the first heading. Let me know what you think.

## Quicksync

When a new node joins the Spacemesh network, it must first get up to speed with the rest of its peers. This process is referred to as "syncing" and is a pre-requisite to running a full or a smeshing node. Historically, it has been difficult for smeshers to successfully sync their nodes owing to how time consuming the syncing process is. Primarily, syncing includes downloading and independently verifying all blocks, transactions, ATXs, along with some other data, and reconstructing the current state from scratch. Naturally, this took the average smesher a lot of time. As such, in response to the growing difficulty of syncing a fresh node, we have prepared a way to speed up the syncing process. Introducing, [Quicksync]((https://spacemesh.io/blog/making-sync-faster/)).

With Quicksync, instead of performing all of the syncing actions as stated above and calculating the network state from genesis, one just needs to download the current state from a trusted peer like the Spacemesh dev team or some other node. While this runs contrary to the web3 philosophy of "Don't trust, verify", we believe that this could be a choice some smeshers may be interested in given the high rate of trouble with syncing. Moreover, nothing precludes a smesher from verifying this state in the background once it is downloaded.

The state (also called an archive) that is downloaded is in the form of a `state.sql` file and can either be downloaded automatically using Smapp, or manually via the CLI. Instructions for both approaches are given below.

### Getting the archive automatically

.... [The guide continues]